packages/core/src/feature-app-manager.ts

```diff
@@ -56,7 +56,7 @@ export interface FeatureAppScope<TFeatureApp> {
   destroy(): void;
 }
 
-export interface FeatureAppScopeOptions {
+export interface FeatureAppScopeOptions<TConfig> {
```

Comment:
Maybe it makes sense that the FeatureServices can also be overwritten? Useful for typed access.

Reply:
For `FeatureAppScopeOptions` which is only used (internally more or less) by the `FeatuerAppContainer` this does not really matter. You could argue though that this might make sense for `FeatureAppContainer`/`FeatureAppContainerProps`. But in contrast to `TConfig` where the type of the `config` prop matches the config property of `env`, the type of the Feature Services in `env` has no counterpart in `FeatureAppContainerProps`. Therefore a type param wouldn't be any better than a type cast. See also https://twitter.com/SeaRyanC/status/1118634571564630016

Reply:
Nevermind, see c01c1f6.